.github/workflows/update-genexus-dep-version.yml

```
@@ -55,11 +55,18 @@ jobs:
                 ref: ${{ github.ref }}
                 fetch-depth: 1
                 token: ${{ github.token}} # Must not use steps.app-token.outputs.token here because it is not allowed to access the current repository
-  
+
+            # Get a non empty value for the commit message 
             - name: Get last commit message
               if: env.COMMIT_MESSAGE == ''
               run: echo "COMMIT_MESSAGE=$(git log -1 --pretty=%B)" >> $GITHUB_ENV
 
+            - name: Normalize Commit Message
+              run: |
+                # Remove ' character from message
+                normalizedMessage=$(echo "${{ env.COMMIT_MESSAGE }}" | sed "s/'//g")
```

```suggestion
                normalizedMessage=$(echo "${{ env.COMMIT_MESSAGE }}" | sed "s/'/\"/g")
```

I would replace the single quote with a double quote char to avoid potentially losing the meaning of any quoted text in the original message.

---

👍